.meteor/packages

```diff
@@ -96,3 +96,4 @@ johanbrook:publication-collector
 
 
 # Custom Packages
+natestrauser:select2
```

Comment:
This stuff will be converted to react soon enough, so probably doesn't matter either way. And the less npm stuff we add that will be removed later, the better.

Also we have a React select / multiselect component in the UI package.